cardinal/system/system_tokeniser.go

```diff
@@ -100,6 +108,14 @@ func (ts *TokeniserSystem) setupVrbAct() {
 	ts.responseLookup[enums.ActionTypeBreak] = []enums.ActionType{enums.ActionTypeBreak}
 }
 
+// setupGrammar initializes the grammar response tookup table with predifined grammar
+func (ts *TokeniserSystem) setupGrammar() {
+	ts.grammarLookup["The"] = enums.GrammarTypeDefinitionArticle
+	ts.grammarLookup["To"] = enums.GrammarTypePreposition
+	ts.grammarLookup["at"] = enums.GrammarTypePrepo
+	ts.grammarLookup["Around"] = enums.GrammarTypeAdverb
+}
+
 // FishTokens processes the tokenized command and returns VerbData
 func (ts *TokeniserSystem) FishTokens(tokens []string) component.VerbData {
```

Comment:
I think this logic is wrong, we arent just looking for a matching value but also its position in the command string, put some tests round it pull some command string from chat gpt then run the fish tokens againt it and make sure that if fishes the VRB, DOBJ and IOBJ and returns them. check the logic in solidity

Reply:
i.e "KICK the ball at the Nasty goblin", "OPEn the door with the RUSTY NAIL" type strings: the kick would give you VRB: KICK DOBJ:BALL, IOBJ: GOBLIN

Reply:
Replaced the FishTokens function on the new commit